doc/API.md

```diff
@@ -360,9 +503,236 @@ Function: Submit a header that has passed the POW.
 
 Parameters:
 ```
-[]byte
+input []byte
+```
+The input byte array should be 80 bytes that form the solved block header. *Unlike most API calls, it should be written directly to the request body, not as a query parameter.*
+
+Renter
+------
+
+Queries:
+
+* /renter/downloadqueue     [GET]
+* /renter/load              [POST]
+* /renter/loadascii         [POST]
+* /renter/rename            [POST]
+* /renter/files             [GET]
+* /renter/delete/<path>     [POST]
+* /renter/download/<path>   [GET]
+* /renter/share/<path>      [GET]
+* /renter/shareascii/<path> [GET]
+* /renter/upload/<path>     [POST]
+
+#### /renter/downloadqueue [GET]
+
+Function: Lists all files in the download queue.
+
+Parameters: none
+
+Response:
+```
+struct {
+	downloads []struct {
+		nickname    string
+		destination string
+		filesize    uint64
+		received    uint64
+		starttime   Time (string)
+	}
+}
+```
+'nickname' is the nickname given to the file when it was uploaded.
+
+'destination' is the path that the file was downloaded to.
+
+'filesize' is the size of the file being download.
+
+'received' is the number of bytes downloaded thus far.
+
+'starttime' is the time at which the download was initiated.
+
+#### /renter/files
+
+Function: Lists the status of all files.
+
+Parameters: none
+
+Response:
+```
+struct {
+	files []struct {
+		nickname       string
+		filesize       uint64
+		available      bool
+		uploadprogress float32
+		expiration     types.BlockHeight (uint64)
+	}
+}
+```
+'nickname' is the nickname given to the file when it was uploaded.
+
+'filesize' is the size of the file in bytes.
+
+'available' indicates whether or not the file can be downloaded immediately.
+
+'uploadProgress' is the current upload percentage of the file, including
+redundancy. In general, files will be available for download before
+UploadProgress == 100.
+
+'expiration' is the block height at which the file ceases availability.
+
+#### /renter/load [POST]
+
+Function: Load a .sia file into the renter.
+
+Parameters:
+```
+filename string
+```
+'filename' is the filepath of the .sia file that is being loaded.
```

Comment:
it should probably be `filepath`, since we're looking up the filepath of a .sia that's being loaded. I don't think 'destination' makes any sense.